gateway/modules/eventing/helpers.go

```diff
@@ -179,6 +179,23 @@ func (m *Module) generateProcessedEventRequest(eventID string) *model.UpdateRequ
 	}
 }
 
+func (m *Module) triggerDLQEvent(ctx context.Context, eventDoc *model.EventDocument) error {
+	req := model.QueueEventRequest{
+		Type: fmt.Sprintf("%s%s", utils.DLQEventTriggerSuffix, eventDoc.RuleName),
+		Payload: map[string]interface{}{
+			"id":   eventDoc.ID,
```

Comment:
Add the following fields to the payload
- prevent event ts
- previous trigger name